test/multicluster/minikube_base_multicluster_test.go

```diff
@@ -0,0 +1,109 @@
+package multicluster
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/nuodb/nuodb-helm-charts/v3/test/testlib"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/random"
+)
+
+func verifyNuoSQL(t *testing.T, namespaceName string, adminPod string, databaseName string) {
+	output, err := testlib.RunSQL(t, namespaceName, adminPod, databaseName, "select * from system.nodes")
+	require.NoError(t, err, output)
+	require.True(t, strings.Contains(output, "Storage"))
+	require.True(t, strings.Contains(output, "Transaction"))
+}
+
+func TestKubernetesBasicMultiCluster(t *testing.T) {
+	if os.Getenv("NUODB_LICENSE") != "ENTERPRISE" && os.Getenv("NUODB_LICENSE_CONTENT") == "" {
+		t.Skip("Cannot test multiple SMs without the Enterprise Edition")
+	}
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+
+	// For multi-cluster deployment to work correctly, there are two prerequisites:
+	// - the same namespace name should be used in all clusters
+	// - the same admin Helm release should be used in all cluster (probably something to fix)
+	randomSuffix := strings.ToLower(random.UniqueId())
+	adminReleaseName := fmt.Sprintf("admin-%s", randomSuffix)
+	namespaceName := fmt.Sprintf("kubernetesbasicmulticluster-%s", randomSuffix)
+
+	options := helm.Options{
+		SetValues: map[string]string{
+			"database.sm.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"admin.readinessTimeoutSeconds":         "5",
```

Comment:
can you provide a comment why this timeout needs to be bumped and what type of issues do we see otherwise? Should we add this knowledge to some runbook?

Reply:
Good point!
This is a leftover from my local testing with K3d multi-cluster. I will remove it.